ui/src/components/EntitySearch/EntitySearchResultsRow.jsx

```diff
@@ -84,39 +105,15 @@ class EntitySearchResultsRow extends Component {
               <Checkbox checked={isSelected} onChange={() => updateSelection(entity)} />
             </td>
           )}
-          <td key="entity" className="entity">
-            <Entity.Link
-              preview={showPreview}
-              documentMode={documentMode}
-              entity={entity}
-              icon
-            />
-          </td>
-          {!hideCollection
-            && (
-              <td key="collection" className="collection">
-                <Collection.Link preview collection={entity.collection} icon />
-              </td>
-            )
-          }
-          {!documentMode && (
-            <td key="country" className="country">
-              <Country.List codes={entity.getTypeValues('country')} />
-            </td>
-          )}
-          <td key="date" className="date">
-            <Date.Earliest values={entity.getTypeValues('date')} />
-          </td>
-          {documentMode && (
-            <td key="file-size" className="file-size">
-              <FileSize value={entity.getFirst('fileSize')} />
-            </td>
-          )}
+          {columns.map(column => {
+            const content = this.renderCellContent(column);
+            return <td key={column.name} className={column.type?.name || column.name}>{content}</td>
+          })}
         </tr>
         {!!highlights.length
           && (
             <tr key={`${entity.id}-hl`} className={highlightsClass}>
-              <td colSpan="5" className="highlights">
+              <td colSpan="100%" className="highlights">
```

Comment:
I think colspan should be all lowercase. Also, colspan? Is there a reason why we wouldn't just have width: 100%; in the css?

Reply:
In jsx I think it does have to be "colSpan" not "colspan" or else the compiler will throw a warning - I could be wrong about this though.

And in this case I feel colspan is favorable as the cleanest way to make a table cell span the width of the table